chanjo/cli/base.py

```diff
@@ -59,7 +59,12 @@ def root(context, config, database, log_level, log_file):
             context.obj = ruamel.yaml.safe_load(conf_handle)
     else:
         context.obj = {}
+
     context.obj['database'] = (database or context.obj.get('database'))
+    
+    context.obj['backend'] = 'sql'
+    if (context.obj['database'] and 'mongo' in context.obj['database']):
```

Comment:
Maybe the type of backend is something that should passed in the config, rather than searching the substring 'mongo' in the uri?

Reply:
Yes I think you are right